homeassistant/components/totalconnect/config_flow.py

```diff
@@ -4,12 +4,10 @@
 import voluptuous as vol
 
 from homeassistant import config_entries
-from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.const import CONF_LOCATION, CONF_PASSWORD, CONF_USERNAME
```

Comment:
I think it can be dropped from the `const.py` file also. It seems unused there.
https://github.com/home-assistant/core/blob/76103752b8cb8479ee507fedddf615644535078b/homeassistant/components/totalconnect/const.py#L5